lib/generators/curation_concerns/install_generator.rb

```diff
@@ -104,5 +105,12 @@ def resource_types_config
     def simple_form_bootstrap
       generate 'simple_form:install --bootstrap'
     end
+
+    def simple_form_initializers
+      copy_file 'config/initializers/simple_form.rb',
```

Comment:
How do these differ from the defaults distributed by simple_form? Why do we need them?